src/main/java/rtg/world/gen/ChunkProviderRTG.java

```diff
@@ -874,16 +970,18 @@ public void populate(IChunkProvider ichunkprovider, int chunkX, int chunkZ)
         if (rand.nextInt(100) == 0) {
     		for(int i19 = 0; i19 < 20; i19++)
     		{
-    			int i22 = worldX + rand.nextInt(16) + 8;
+    			int i22 = worldX + rand.nextInt(16);// + 8;
     			int l23 = rand.nextInt(worldHeight / 2);
-    			int i25 = worldZ + rand.nextInt(16) + 8;
+    			int i25 = worldZ + rand.nextInt(16);// + 8;
     			(new WorldGenLiquids(Blocks.flowing_lava)).generate(worldObj, rand, i22, l23, i25);
     		}
         }
 
+        TimeTracker.manager.stop("Post-decorations");
+        TimeTracker.manager.start("Entities");
         if (TerrainGen.populate(this, worldObj, rand, chunkX, chunkZ, flag, PopulateChunkEvent.Populate.EventType.ANIMALS))
         {
-            SpawnerAnimals.performWorldGenSpawning(this.worldObj, worldObj.getBiomeGenForCoords(worldX + 16, worldZ + 16), worldX + 8, worldZ + 8, 16, 16, this.rand);
+            SpawnerAnimals.performWorldGenSpawning(this.worldObj, worldObj.getBiomeGenForCoords(worldX + 16, worldZ + 16), worldX, worldZ, 16, 16, this.rand);
         }
 
```

Comment:
Oh, yeah, I know, I just said that the TimeTracker for entities should `stop` just below this line, and not below the Ice gen (maybe Github didn't show it for you, you have to press the button on the left in the diff page or just use the IDE). In other words, the tracker for Entities also counts the time spent on Ice gen and that shouldn't happen obviously.